.expeditor/verify.pipeline.yml

```diff
@@ -12,10 +12,10 @@ expeditor:
 steps:
 
 #########################################################################
-  # Tests Ruby 2.6
+  # Tests Ruby 2.7
```

Comment:
chef-utils and chef-config were still on ruby 2.6 by design. These tests need to stay